main.py

```diff
@@ -846,6 +849,15 @@ def patch_record_in_place(fn, record, subdir):
     # anaconda, conda, conda-build, constructor #
     #############################################
 
+    # Remove old shortcut packages from Navigator panel
+    if name.startswith("console_shortcut") or name.startswith("powershell_shortcut"):
```

Comment:
```suggestion
    if name in ["console_shortcut", "console_shortcut_miniconda", "powershell_shortcut", "powershell_shortcut_miniconda"]:
```
Just a suggestion, I find this more explicit.